mongodb-atlas/cmd/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/icholy/digest"
+	//mongodbatlasplugin "github.com/opencost/opencost-plugins/mongodb-atlas/mongodbatlasplugin"
+)
+
+// TODO move these types to mongodbatlasplugin package
+type CreateCostExplorerQueryPayload struct {
+	Clusters              []string `json:"clusters"`
+	EndDate               string   `json:"endDate"`
```

Comment:
Ive modified the function itself to take time.Time.  So its more strict. Internally we are creating this struct.  when I tried changing EndDate to time.Time issue I was running into was that the string was of the format yyyy-mm-dd+timestamp and mongo atlas didn't like it. Instead of figuring out how to modify the json.Marshal to marshal a time type to yyyy-mm-dd without the timestamp I just do a parse in the function itself.